frontend/src/lib/components/TaxonomicFilter/InfiniteList.tsx

```diff
@@ -400,7 +400,7 @@ export function InfiniteList(): JSX.Element {
                             height={height}
                             rowCount={isLoading && totalListCount === 0 ? 7 : totalListCount}
                             overscanRowCount={100}
-                            rowHeight={32}
+                            rowHeight={36} // LemonRow heights
```

Comment:
Increasing to 40px as they look like in the mockups took up way too much space. Settled at a more reasonable 36.